### ISSUE TITLE
[Feature Request] Let the user decide if the videos are auto marked as watched or not

### ISSUE THREAD

**Issue by Zhyrclew** [CodeNinjaUK/youtube-subs-grid#87]
**Add an option to desactivate the auto mark as watched when you click on a video and watch it.**

I'm pretty sure every user of your extention who is subscribed to YT channels publishing music has been bothered by this :
- You have a big subscription feed, so you have to hide the watched video, in order to have a visual on more than one day of subscription ;
- You have a new music video on your feed, you watch it, you find it good. Real good. You say, i wanna hear this again, in a few hours or days, in order to decide if i'll add it to a playlist, or (legally) download it. The video has already vanished from your feed, because of the hide watched videos option.
- You have to show the watched videos, hopefully the music video was a recent one, so you find it and unmark it as watched to be able to see it again when you'll hide the watched videos, and keep an eye on it for the next week or so.
- The next day, you see again this music video, no music playing in your headset, you say, oh i want to hear it again ! Click on it, hear it, sound good, real good... Go to your subscription grid; vanished again.  Damn it ! Lost it forever and ever in the chaos of watched videos ! Noooooo !
- The above point, again, and again, and again... *BANG*

Well, maybe not all of them killed themself because of it, but i'm about to, and if you don't make soon a option to desactivate this damned auto mark as watched you'll have my death on your hands...

**Comment by Droyk** [comment]
Same as #82

**Comment by Zhyrclew** [comment]
No, "Let The User Decide If They Want To Auto Hide The Watched Videos Or Not" is not what I asked for. 
Yes I read your request before posting mine, and the option not to auto hide watched videos would not be adequate for my use : 
I want the videos I mark as watched to be hidden right as I mark them. To disable the fact that watched videos are automatically hidden would solve the issue that I reported but it would create another big issue as I need to auto hide the videos I mark myself as watched, to keep my feed clear (as pointed out in my first point above).
So the option you requested is not the one I requested here, even if your option solve part of my issue. (and create one more, in my case)
As you said that you write bad english, maybe you didn't wanted to resquest the option you described, in that case i feel usefull to create another post with a more accurate description. If not, well we are not requesting the same feature.

**Comment by DanSmith70** [comment]
This is a tough one to solve because once you watch a video it's actually YT that mark it as watched, and there's no way for me to override that. The only way I can think to fix this in the extension would be add some kind of featured or favourite option so you can come back to it later. 

That said, have you considered hitting the "watch later" button which will add the video to your watch later playlist?

**Comment by Zhyrclew** [comment]
Oh right that's problematic... But i'm a little bit surprised to hear that, because the videos I watch when the extension is desactivated aren't in your watched videos list when I reactivate it.
So there must be a way to differentiate between videos added by YT by watching them, and videos added by clicking on the watched button from subgrid, especially since you have the control of the watch list, am i wrong ?
Yeah, no the "watch later" is realy not adapted for that particular use, trust me. Plus, your extention doesn't make watchlists into grids, so it's not even easily readable.
PS: I remember a period when your extention was having troubles to adapt to YT changes, and where the button "mark as watched" was still working, but the auto mark as watched when viewing the video wasn't  working !
